packages/sommelier.rb

```diff
@@ -374,58 +440,61 @@ def self.install
   end
 
   def self.postinstall
-    # all tasks are done by sommelier.env now
-    now = Time.now.strftime('%Y%m%d%H%M')
-    FileUtils.cp "#{HOME}/.bashrc", "#{HOME}/.bashrc.#{now}"
-    system "sed -i '/[sS]ommelier/d' #{HOME}/.bashrc"
-    diff = `diff #{HOME}/.bashrc #{HOME}/.bashrc.#{now}`.chomp
-    if diff == ''
-      FileUtils.rm "#{HOME}/.bashrc.#{now}"
+    if ARCH == 'i686'
+      puts 'Note that sommelier is only a nonfunctional stub package on i686.'.lightred
```

Comment:
Not needed if we just abort from the start?  Also, we can just take it out of `compatibility`.